src/SonarQube.Client.Tests/Api/SonarQubeService_GetNotificationEventsAsync.cs

```diff
@@ -100,6 +101,12 @@ public async Task GetNotifications_NotFound()
             result.Should().BeNull();
 
             messageHandler.VerifyAll();
+
+            logger.DebugMessages.Should().Contain(
+                new[]
+                {
+                    "Notifications not enabled on SonarQube. You would need at least Developer Edition to use this feature.",
```

Comment:
Minor: I'd drop _would_ from this sentence, or use an alternative wording like
_This features requires Developer Edition or higher._